GSA-TTS/FAC
# Update Primary Nav to Match the Static Site

# Update Primary Nav to Match the Static Site

No issue, quick fix.

## Change:
1. Update the "Helpdesk" link in the primary nav to "Contact"
2. Switch "Developer resources" and "Data reliability"
3. Add "System status"

## How to test:
Switch to this branch, run normally. Verify the updated links look correct and work properly. 


## PR Checklist: Submitter


## PR Checklist: Reviewer


The larger the PR, the stricter we should be about these points.

## Pre Merge Checklist: Merger
